controller/GraphQLController.java

```diff
@@ -50,55 +56,116 @@ public GraphQLController(ConfigurationDAO config, BentoGraphQL bentoGraphQL){
 	@CrossOrigin
 	@RequestMapping(value = "/version", method = {RequestMethod.GET},
 			produces = MediaType.APPLICATION_JSON_VALUE + "; charset=utf-8")
-	public ResponseEntity<String> getVersion(HttpEntity<String> httpEntity, HttpServletResponse response){
-		logger.info("Hit end point:/version");
+	public ResponseEntity<String> getVersion(){
+		logger.info("Hit end point: /version");
 		String versionString = config.getBentoApiVersion();
 		logger.info(versionString);
 		return ResponseEntity.ok(gson.toJson(Map.of("version", versionString)));
 	}
 
+	@CrossOrigin
+	@RequestMapping(value = "/neo4j-version", method = {RequestMethod.GET},
+			produces = MediaType.APPLICATION_JSON_VALUE + "; charset=utf-8")
+	public ResponseEntity<String> getNeo4jVersion(HttpEntity<String> httpEntity){
+		logger.info("Hit end point: /neo4j-version");
+		try{
+			//Build URI and GraphQL query
+			URI uri = new URI(config.getNeo4jUrl());
+			String neo4jVersionQuery = "{\"query\":\"{neo4jVersion}\",\"variables\":{}}";
+			//Create request HttpEntity
+			httpEntity = new HttpEntity<>(neo4jVersionQuery, httpEntity.getHeaders());
+			//Send request
+			ResponseEntity<String> responseEntity = getGraphQLResponse(httpEntity, bentoGraphQL.getPublicGraphQL());
+			JsonObject jsonResponseBody = gson.fromJson(responseEntity.getBody(), JsonObject.class);
+			if (jsonResponseBody.has("errors")){
+				return responseEntity;
+			}
+			//Parse version from response
+			String version = gson.fromJson(responseEntity.getBody(), JsonObject.class)
+					.get("data").getAsJsonObject()
+					.get("neo4jVersion").getAsString();
+			//Format and return info
+			return ResponseEntity.ok(gson.toJson(new DataSourceVersion(uri, version)));
+		}
+		catch (Exception e){
+			return logAndReturnError(HttpStatus.INTERNAL_SERVER_ERROR,
+					"An internal server error occurred, please contact the system custodians", e);
+		}
+	}
+
+	@CrossOrigin
+	@RequestMapping(value = "/opensearch-version", method = {RequestMethod.GET},
+			produces = MediaType.APPLICATION_JSON_VALUE + "; charset=utf-8")
+	public ResponseEntity<String> getOpenSearchVersion(){
+		logger.info("Hit end point: /opensearch-version");
+		try{
+			//Build URI
+			URIBuilder builder = new URIBuilder()
+					.setScheme("http")
```

Comment:
This is probably not working on AWS. Please, check whether AWS is using https. Or, you could utilize EsService.send(request) instead.